metals/src/main/scala/scala/meta/internal/metals/InitializationOptions.scala

```diff
@@ -48,6 +48,7 @@ import org.eclipse.{lsp4j => l}
  *                                    copy results to the local buffer.
  * @param disableColorOutput in the situation where your DAP client may not handle color codes in
  *                            the output, you can enable this to strip them.
+ * @param doctorVisibilityProvider if the clients implements `metals/doctorVisibilityDidChange`
```

Comment:
Can we also add this to `new-editor.md` with a note about how it's supposed to be used by a client.